packages/itwinui-css/src/button/button.scss

```diff
@@ -10,6 +10,11 @@
 .iui-button {
   @include base.iui-button;
 
+  // Reusing hover styling when an associated popover/dropdown is open.
+  &:is([data-iui-has-popover='open'], :has(+ :popover-open)) {
```

Comment:
To confirm, what is the `:has(+ :popover-open)` for? Is it a safeguard for when we use the Popover API in more places in the future?

Reply:
Ohh, I see, I was also hoping to understand the DOM for this. Since the floating content will likely be portaled to the `ThemeProvider`, will this sibling selector have any effect?

Reply:
It should not be portaled. We kept the portal implementation in tooltip, but ideally it should be a sibling.

```html
<button popover popovertarget="my-popover">…</button>
<div id="my-popover">…</div>
```

This is important to allow keyboard/AT users to easily navigate back and forth between the trigger and the popover, without needing to write a lot of JS to manage keyboard focus and virtual focus.